src/explorer/pagerankTable/TableRow.js

```diff
@@ -95,3 +106,12 @@ export function PaddingRow(props: {|+backgroundColor: string|}) {
     </tr>
   );
 }
+
+const styles = StyleSheet.create({
+  hoverHighlight: {
```

Comment:
Please document the semantics/contract for `hoverHighlight`—it’s not at
all clear from the name (which is fine in this case: there’s not really
a good name that conveys exactly what the client needs to do).